File: phyton/1113.py
X,Y=list(map(int,input().split(" ")))

while X!=Y:
    if X > Y:
        print("Decrescente")
        X,Y=list(map(int,input().split(" ")))
        
    if X < Y:
        print("Crescente")   
        X,Y=list(map(int,input().split(" ")))

File: c/1113.c
int main ()
{
    int a;
    int b;
    scanf("%d %d", &a, &b);

    while (a != b)
    {
        if (a > b){
            printf("Decrescente");
            scanf("%d %d", &a, &b);
        }
        if (a < b)
        {
            printf("Crescente");
            scanf("%d %d", &a, &b);
        }
        
    }
    return 0;
}